lib/wings/active_fedora_converter.rb

```diff
@@ -107,13 +107,23 @@ def initialize(uri = RDF::Node.new, _parent = ActiveTriples::Resource.new)
     private
 
       def convert_members(af_object)
-        return unless resource.respond_to?(:member_ids) && resource.member_ids
+        return unless resource.respond_to?(:member_ids) && !resource.member_ids.blank?
```

Comment:
```suggestion
        return unless resource.respond_to?(:member_ids) && resource.member_ids.present?
```
This would avoid double negation (https://api.rubyonrails.org/classes/Object.html#method-i-present-3F)